components/webdriver_server/lib.rs

```diff
@@ -1110,7 +1111,12 @@ impl Handler {
         let prefs = parameters
             .prefs
             .iter()
-            .map(|item| (item.clone(), serde_json::to_value(PREFS.get(item)).unwrap()))
+            .map(|item| {
+                (
+                    item.clone(),
+                    serde_json::to_value(prefs::pref_map().get(item)).unwrap(),
```

Comment:
Is it worth adding a method to retrieve multiple prefs at once, to avoid locking and unlocking multiple times?

Reply:
As mentioned in other comments, we can restructure the internals in a later PR, so that properties are locked individually and use atomics where appropriate.